server/src/main/java/com/broadcom/lsp/cobol/core/model/variables/UsageFormat.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+
+package com.broadcom.lsp.cobol.core.model.variables;
+
+import java.util.stream.Stream;
+
+/**
+ * This enum class represents all the supported usage formats. UNDEFINED means that the usage format
+ * cannot be determined.
+ */
+public enum UsageFormat {
+  BINARY,
+  COMP,
+  COMP_1,
+  COMP_2,
+  COMP_3,
+  COMP_4,
+  COMP_5,
+  COMPUTATIONAL,
+  COMPUTATIONAL_1,
+  COMPUTATIONAL_2,
+  COMPUTATIONAL_3,
+  COMPUTATIONAL_4,
+  COMPUTATIONAL_5,
+  DISPLAY,
+  DISPLAY_1,
+  INDEX,
+  NATIONAL,
+  UTF_8,
```

Comment:
Optional:  Sort by 🔠 by IntelliJ String Manipulation plugin

Reply:
I wanted to have them in the same order as in the specification for better consistency